compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -1208,7 +1208,9 @@ def _on_message(
         message to AMQP service -- the responsibility to handle invalid messages
         is not with this class.
         """
-        msg_id: int = basic_deliver.delivery_tag
+        msg_id: int = 0  # TODO throw an exception if this is missing?
+        if basic_deliver.delivery_tag is not None:
+            msg_id = basic_deliver.delivery_tag
```

Comment:
From the AMQP protocol, this is a problem if it's missing so I don't expect that to fail.  If it does, I want it to fail loudly so we can address it correctly and precisely to the protocol.  What are you noticing and/or what motivated this change?

Reply:
Originally, mypy or some lint was complaining about the msg_id or the type.  Reverted and perhaps it'll now work.